jenkins/node-image/Makefile

```diff
@@ -15,14 +15,39 @@
 IMG = gcr.io/k8s-testimages/kubekins-node
 TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 
+# default to 1.6 image
+VERSION:=1.6
+KUBEKINS_TAG:=1.6-v20161205-ad918bc
```

Comment:
another thought: add `VERSION-latest` tags to the kubekins-test image, and then run `docker build --pull` to make sure we're actually using the latest version? then we don't need to specify an explict tag here in the `Makefile`.

I'm not sure if there are any issues with that.